CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.1.0-beta.2
+- Interact now returns `IdxContext` struct
+- Introspect requires a second parameter, `state *string`. This can be `nil` if you want the library to create a state for you.
+
+- `IdxContext.interactionHandle` stores a string for the interaction handle. Can be retrieved with `IdxContext.GetInteractionHandle()`
+- `IdxContext.state` stores the a state string. This can be set during `Interact` as the second parameter, and can be retrieved with `IdxContext.GetState()`
```

Comment:
```suggestion
- `IdxContext.state` stores the state string. This can be set during `Interact` as the second parameter, and can be retrieved with `IdxContext.GetState()`
```